src/wordlift/mappings/class-mappings-dbo.php

```diff
@@ -55,27 +55,37 @@ public function get_mappings() {
 	public function get_active_mappings() {
 		$mapping_table_name = $this->wpdb->prefix . WL_MAPPING_TABLE_NAME;
 
-		return $this->wpdb->get_results(
+		$mappings = $this->wpdb->get_results(
 			"SELECT * FROM $mapping_table_name WHERE mapping_status = 'active'",
 			ARRAY_A
 		);
+
+		return apply_filters( 'wl_mappings_register_mappings', $mappings );
```

Comment:
@naveen17797 pls add PhpDoc here. I would also rename the filter `wl_mappings_active_mappings` to reflect that this list is about `active` mappings.